imagery/i.svm.train/fill.c

```diff
@@ -0,0 +1,112 @@
+/*
+ * i.svm.train Functions filling svm_problem struct
+ *
+ *   Copyright 2020 by Maris Nartiss, and The GRASS Development Team
+ *   Author: Maris Nartiss
+ *
+ *   This program is free software licensed under the GPL (>=v2).
+ *   Read the COPYING file that comes with GRASS for details.
+ *
+ */
+#include <grass/raster.h>
+#include <grass/glocale.h>
+
+#include "fill.h"
+
+void fill_problem(const char *name_labels, const char *mapset_labels,
+                  struct Ref band_refs, const char *mapset_group,
```

Comment:
```suggestion
                  struct Ref band_refs, const char *mapset_group UNUSED,
```

or remove the parameter.

Reply:
Why this was not spotted by cppcheck or clang-tidy? Removed.